package.json

```
@@ -117,6 +117,7 @@
     "react-native-mmkv": "2.10.2",
     "react-native-os": "https://github.com/aprock/react-native-os#938676ebd92d58b8928168a7e78cbe216838ed17",
     "react-native-permissions": "^3.10.0",
+    "react-native-popup-menu": "^0.16.1",
```

Please fix the version and recreate lock file
```suggestion
    "react-native-popup-menu": "0.16.1",
```

---

Done